src/gui/widgets/window.cpp

```diff
@@ -1243,8 +1243,8 @@ void swap_grid(grid* g,
 		parent_grid = find_widget<grid>(content_grid, id, true, false);
 		assert(parent_grid);
 	}
-	if(grid* g = dynamic_cast<grid*>(parent_grid->parent())) {
-		widget = g->swap_child(id, widget, false);
+	if(grid* g_in_if = dynamic_cast<grid*>(parent_grid->parent())) {
```

Comment:
This is a really terrible name. I think I'd prefer renaming the parameter instead in this case. Not entirely sure what's a good name though. Maybe the doxy-comment in the header can give a good idea.